lib/cmd/fh3/services/list.js

```diff
@@ -16,13 +16,10 @@ module.exports = {
   'describe' : {
     'json' : i18n._('Output into json format')
```

Comment:
👍 I create the JIRA for this: https://issues.jboss.org/browse/FH-3650
Please, feel free to contribute with or thoughts there.